pkg/provider/ngrok/provider_test.go

```diff
@@ -0,0 +1,219 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or impliec.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ngrok
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubeClient "sigs.k8s.io/controller-runtime/pkg/client"
+	clientfake "sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	esv1 "github.com/external-secrets/external-secrets/apis/externalsecrets/v1"
+	v1 "github.com/external-secrets/external-secrets/apis/meta/v1"
+)
+
+func TestProviderCapabilities(t *testing.T) {
+	provider := &Provider{}
+	capabilities := provider.Capabilities()
+
+	assert.NotNil(t, capabilities)
+	assert.Equal(t, esv1.SecretStoreWriteOnly, capabilities)
+}
+
+func TestProviderValidateStore(t *testing.T) {
```

Comment:
These tests don't seem to test anything. 🤔  Or rather, just the mock really. I changed some values here and there and nothing broke. :D For example "valid store" just does nothing really. 🤔